src/main/kotlin/uk/gov/gdx/datashare/config/OpenApiConfiguration.kt

```diff
@@ -30,11 +32,27 @@ class OpenApiConfiguration(
         Server().url("/").description("Current url"),
       ),
     )
+    .tags(
+      listOf(
+        Tag().name("01. Acquirer").description("Acquiring events from the platform"),
+        Tag().name("02. Supplier").description("Supplying events to the platform"),
+        Tag().name("11. Acquirers").description("Maintenance of Acquirers"),
+        Tag().name("12. Suppliers").description("Maintenance of Suppliers"),
+        Tag().name("13. Admin").description("Admin functions"),
+        Tag().name("20. Data").description("Maintenance of Data"),
```

Comment:
I wonder if we can do some clever reflection so we don't need to duplicate this list